clusters/hypershift-ci-1/manifests/hypershift-imagestream.yaml

```diff
@@ -15,6 +15,14 @@ spec:
       type: Local
     importPolicy:
       scheduled: true
+  - name: ci
+    from:
+      kind: DockerImage
+      name: quay.io/hypershift/hypershift-operator:e43a80bb
```

Comment:
any reason not to use a sha256 tag?

Reply:
[quay.io](http://quay.io/) does not preserve tag history so without an actual tag, the image will get pruned out of the repo. The other reason is that it's easy to figure out what commit we have running (the tag is the commit hash)